concept/thing/statistics.rs

```diff
@@ -167,6 +175,14 @@ impl Statistics {
             self.durably_write(storage.durability())?;
         }
 
+        let millis = Instant::now().duration_since(start).as_millis();
+        event!(
+            Level::TRACE,
+            "Statistics sync finished in {} ms. Storage watermark was initially: {}. Current statistics sequence is from: {}",
+            millis,
+            storage_watermark,
+            start_seq_nr
+        );
```

Comment:
Really useful when investigating whether statistics is able to keep up with the current 100ms update interval @dmitrii-ubskii  @cxdorn